utils/issuescollection.go

```diff
@@ -54,3 +57,20 @@ func (ic *IssuesCollection) Append(issues *IssuesCollection) {
 		ic.Licenses = append(ic.Licenses, issues.Licenses...)
 	}
 }
+
+func (ic *IssuesCollection) CountIssuesCollectionFindingsForAnalytics() int {
```

Comment:
```suggestion
func (ic *IssuesCollection) CountIssuesCollectionFindings() int {
```
Method is counting the issues, no need to mention yhe usage of it since it could be usefull in many different logics